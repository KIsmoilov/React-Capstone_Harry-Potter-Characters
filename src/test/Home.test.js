import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Characters from '../pages/HomePage';
import store from '../redux/configureStore';

it('Builds the snapchot of the Home component Details', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <Router>
          <Characters />
        </Router>
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
