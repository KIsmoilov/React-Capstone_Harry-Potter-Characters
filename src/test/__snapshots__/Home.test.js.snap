// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Builds the snapchot of the Home component Details 1`] = `
<div
  className="home-contain"
>
  <div
    className="container"
  >
    <h1>
      Harry Potter Characters
    </h1>
    <div
      className="input"
    >
      <input
        className="inputField"
        onChange={[Function]}
        placeholder="Search by name..."
        value=""
      />
    </div>
  </div>
  <div
    className="characters-main-section"
  />
</div>
`;
